manuscript/main.tex

```diff
@@ -1046,8 +1060,21 @@ \subsubsection*{Machine learning for cognitive development}
 representations of the gut microbiome (taxonomic profiles, functional
 profiles encoded as ECs) would behave, with and without the addition of
 demographics (sex and maternal education as a proxy of socioeconomic
-status) on participants from different age groups. Age (in months) was
-provided as a covariate for all models (Table S3).
+status) on participants from different age groups.
+
+For the concurrent cognitive assessment score preditions,
+all viable datapoints containing 
+age at data collection, demographics, stool metagenomics and cognitive assessment were employed.
```

Comment:
```suggestion
For the concurrent cognitive assessment score predictions,
we included all visits where age at data collection, demographics (maternal education, %anything else?
), stool metagenomics and cognitive assessment were collected.
For subjects that had multiple visits matching the selection criteria, the % first? last?
visit was used.
```

Reply:
Should I remove lines 1071-1072, "For the concurrent prediction experiments, when multiple viable longitudinal samples were present, the single earliest viable datapoint inside the age range of the experiment was considered.">> This makes them redundant imho

Reply:
Either way